charts/zac/README.md

```diff
@@ -130,7 +130,7 @@ A Helm chart for installing Zaakafhandelcomponent
 | nginx.existingConfigmap | string | `nil` | mount existing nginx vhost config |
 | nginx.image.pullPolicy | string | `"IfNotPresent"` |  |
 | nginx.image.repository | string | `"nginxinc/nginx-unprivileged"` |  |
-| nginx.image.tag | string | `"1.26.2@sha256:4dc24adb9f46f6c5bbdea47e03a22343d515687c1c9077cf3df36dd9c0a97441"` |  |
+| nginx.image.tag | string | `"1.27.3@sha256:d0ebbd60453d566b6c497adb4c0c220ba0aa1bc0d654a6975aa65e16ae11d8fb"` |  |
```

Comment:
Is this upgrade meant to be part of this PR?

Reply:
I had to rebuild the readme because of the changes I did in helm chart (values and config). This one was not rebuilt after renovate PR was merged, so it appears now.

Reply:
tl;dr: not meant to be part of the PR, but rebuild of readme picked up previous changes

Reply:
Removed README changes